tests/unit-tests/console/test_linux_virtual_terminal.cpp

```diff
@@ -777,8 +778,8 @@ void set_expectations_for_uevent_probe(
     std::stringstream expected_filename;
     expected_filename << "/sys/dev/char/" << major << ":" << minor << "/uevent";
 
-    auto uevent = std::make_shared<mir::AnonymousShmFile>(strlen(content));
-    ::memcpy(uevent->base_ptr(), content, strlen(content));
+    auto uevent = std::make_shared<mir::AnonymousShmFile>(mir::strlen_c(content));
+    ::memcpy(uevent->base_ptr(), content, mir::strlen_c(content));
```

Comment:
There's no chance of compile time evaluation here: `content` is not a string literal